resources/js/Components/ItemIdSearchTextarea.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <cdx-field 
+      :status="validationError ? validationError.type : 'default'" 
+      :messages="validationError ? validationError.message : null"
+  >
+      <div class="progress-bar-wrapper">
+          <cdx-progress-bar v-if="loading" :aria-label="$i18n('item-form-progress-bar-aria-label')" />
+      </div>
+      <cdx-text-area
+          :label="$i18n('item-form-id-input-label')"
+          :placeholder="$i18n('item-form-id-input-placeholder')"
+          :rows="8"
+          :status="validationError ? validationError.type : 'default'"
+          v-model="textareaInputValue"
+      />
+  </cdx-field>
+</template>
+
+<script setup lang="ts">
+import { ref, inject } from 'vue';
+import { Ref } from 'vue';
+import { useI18n } from 'vue-banana-i18n';
+import { useStore } from '../store';
+import { CdxTextArea, CdxField, CdxProgressBar } from "@wikimedia/codex";
+import ValidationError from '../types/ValidationError';
+
+// Run it with compat mode
+// https://v3-migration.vuejs.org/breaking-changes/v-model.html
+CdxTextArea.compatConfig = {
+    ...CdxTextArea.compatConfig,
+    COMPONENT_V_MODEL: false,
+};
+
+const validationError: Ref<ValidationError> = ref(null);
+
+const banana = useI18n();
```

Comment:
Are we sure this is the variable name we want to use for this? I'll accept it if the answer is yes, just want to make sure we give this proper consideration

Reply:
the plugin is called `vue-banana-i18n`, so "banana" is not such a strange name for the variable. In the use case we found [in the original library](https://github.com/santhoshtr/vue-banana-i18n/blob/master/src/index.js#L52) they call it `bananai18n`, but it's almost the same.